collections/bitcoin/_posts/2022-10-02-bitcoin-is-digital-scarcity.md

```diff
@@ -1037,8 +1037,8 @@ decentralization and security, nothing else.
 In short: bitcoin can't be printed because it is automatically issued
 over time. The system can't be cheated because energy can't be copied.
 Proof-of-work can't be simulated, and the difficulty adjustment forbids
-the "farming" of blocks. Anyone can participate in, instantiate, and
-validate everything.
+the "farming" of blocks. Anyone can participate, instantiate, and
+validate.
```

Comment:
I don't really like this change, because I the "everything" is kind of important. 

You can participate in everything. You can instantiate everything. And you can validate everything.

Reply:
Agree on instantiate and validate, but you can't really participate in "everything"? You can participate in the Bitcoin network, but you can't practically participate in every aspect of it?

Reply:
You can, and that's the whole point. You can mine, you can generate keys, you can run a node.

Reply:
Ok, I was coming at it from the angle of "participate in every single transaction, mine every block" etc